lib/bugsnag/reporter.ex

```diff
@@ -32,7 +32,7 @@ defmodule Bugsnag.Reporter do
   end
 
   def sync_report(exception, options \\ []) do
-    stacktrace = options[:stacktrace] || System.stacktrace()
+    stacktrace = options[:stacktrace] || __STACKTRACE__
```

Comment:
these need to be passed explicitly from the point where the exception occurs, i guess